content/docs/integrations/launchdarkly/index.md

```diff
@@ -1,59 +1,150 @@
+    launchdarkly.com/context: '{"kind":"tenant","key":"roadie","name":"roadie"}'
+```
+
+#### Advanced Configuration with Filtering
 
+```yaml
+metadata:
+  annotations:
+    launchdarkly.com/project-key: default
+    launchdarkly.com/environment-key: production
+    launchdarkly.com/context: '{"kind":"tenant","key":"roadie","name":"roadie"}'
+    launchdarkly.com/filter-tags: '["frontend", "api"]'
+    launchdarkly.com/filter-query: 'dark-mode'
 ```
+
+For the `EntityLaunchdarklyContextOverviewCard` component `project-key`, `environment-key` and `context` are needed.
+`EntityLaunchdarklyCard` component supports `project-key` and `filter-query`.
+
+
+## Annotation Reference
+
+| Annotation                         | Description                      | Required | Example                                              |
+|------------------------------------|----------------------------------|----------|------------------------------------------------------|
+| `launchdarkly.com/project-key`     | LaunchDarkly project identifier  | Yes      | `default`                                            |
+| `launchdarkly.com/environment-key` | Environment within the project   | Yes      | `production`                                         |
+| `launchdarkly.com/context`         | JSON context for flag evaluation | No       | `'{"kind":"tenant","key":"roadie","name":"roadie"}'` |
+| `launchdarkly.com/filter-tags`     | Array of tags to filter flags    | No       | `'["frontend", "api"]'`                              |
+| `launchdarkly.com/filter-query`    | Query string to filter flags     | No       | `'dark-mode'`                                        |
+
+## Troubleshooting
+
+### Feature flags not appearing
+
+1. **Verify API token configuration**
+   - Check that `LAUNCHDARKLY_API_KEY` is correctly set in Roadie secrets
+   - Ensure the API token has not expired
+   - Verify the token has appropriate read permissions
+
+2. **Check entity annotations**
+   - Confirm `launchdarkly.com/project-key` matches your LaunchDarkly project
+   - Verify `launchdarkly.com/environment-key` exists in your project
+   - Ensure JSON context format is valid if using context annotation
+
+3. **Validate LaunchDarkly configuration**
+   - Confirm the project and environment exist in LaunchDarkly. Note that launchdarkly environment name and the environment key may not be the same.
+   - Check that feature flags exist in the specified environment
+   - Verify network connectivity to LaunchDarkly API
+
+### UI components not displaying
+
+1. **Check component availability**
+   - Verify the LaunchDarkly plugin is enabled in your Roadie instance
+   - Ensure UI components are properly configured on entity pages
+   - Check browser console for JavaScript errors
+
+2. **Validate entity conditions**
+   - Confirm entities have the required LaunchDarkly annotations
+   - Check that the entity type supports the configured UI components
```

Comment:
Does the entity type matter for the ui components?

Reply:
does not matter